test/formats/test_mp4.py

```diff
@@ -81,7 +81,7 @@ def test_replaygain_tags_case_insensitive(self):
         def test_ci_tags_preserve_case(self):
             # Ensure values are not duplicated on repeated save and are saved
             # case preserving.
-            for name in ('Replaygain_Album_Peak', 'Custom', 'äöüéß\0'):
+            for name in ('Replaygain_Album_Peak', "Label", 'Custom', 'äöüéß\0'):
```

Comment:
Minor thing, but please update this to use single quotation marks for consistency.